### ISSUE TITLE
"Error: operation not permitted" when clicking create project (Preparation 2)

### ISSUE THREAD

**Issue by liam-hart** [STA518-01-SS21/community#8]
On preparation 2, I followed the steps in task 1 but there was no README folder. I created a new project, saved it within my new STA 418 folder, named it preparation02, checked _Create a Git repository_, and clicked create project. There was no README folder in preparation02, only preparation02.rproj and .gitconfig. The .gitconfig already had my name and email address because I took this class last semester.
I then deleted the file and went through task 1 again in the same way, thinking I had forgotten to check _Create a Git repository_ or something, but it was already checked when I went through it this time so I must have had it checked before. When I clicked create project this time, it gave me an error saying “operation not permitted” and the only file in preparation02 folder is preparation02.rproj (no .gitconfig or README).

In the .gitconfig, my name and email address were already input from last semester, so I am thinking it is a problem with that, but I don't know how to fix this error.

**Comment by shahnikcode** [comment]
It did not work for me either.  Im not able to access git through r server still and i don't get prompted to put in a PAT.  I tried searching the error and downloading the recommended r packages to see if that would help and i can't access them.  Do you have any other ideas or is this something we can troubleshoot in class tomorrow?

**Comment by dykesb** [comment]
> On preparation 2, I followed the steps in task 1 but there was no README folder. 

I just realized I gave conflicting directions here, I am sorry about that. Since you are creating a repo from scratch, you should not see the `README-img/` folder. Note that this folder IS in the Preparation 2 directions repo.

> When I clicked create project this time, it gave me an error saying “operation not permitted”. 

Did you delete your entire `preparation02` folder? Specifically, that is

- Click on the big blue R (upper-left corner), quit all sessions, start a new session
- In the Files pane (lower-right pane), navigate to the preparation02 folder previous created (do not go into the folder)
- Check the box next to it, click on Delete in this pane's tool bar
- Click Yes on the Confirm Delete pop-up

If not, it might be an issue that you are trying to create two things named the same thing.

Otherwise, try the GitHub First method outlined in [this other Issue](https://github.com/STA518-01-SS21/community/issues/7#issuecomment-838891521)

**Comment by liam-hart** [comment]
I tried both methods you gave and came up with more errors. When trying to delete the preparation02 folder, it now gives me an error that says "Device or resource busy". When I try to use Github first and clone it using the HTTPS link after selecting version control and then Git, it says "fatal: bad config line 2 in file /home/hartli@hpc.gvsu.edu/.gitconfig" and does not clone the repository. From here, I tried typing "git config --global user.name 'Liam Hart'" and "git config -- global user.email 'hartli@mail.gvsu.edu'" but it gives me the same "fatal" message. Not sure where to go from here.
